File: src/test/java/com/mako/StudentTest.java
package com.mako;

import com.mako.student.Student;
import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class StudentTest {
    @Test
    void highestScoreTest() {
        Student student = new Student("Mako", 70, 90);
        Assertions.assertEquals(90, student.highest());
    }

    @Test
    void averageTest() {
        Student student = new Student("MAKO", 60, 100);
        Assertions.assertEquals((60+100)/2, student.getAverage());
    }
}

File: src/test/kotlin/com/mako/StudentTest.kt
package com.mako.kotlin

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

class StudentTest {
    @Test
    fun highestTest() {
        val student = Student("Mako", 70, 100)
        Assertions.assertEquals(100, student.highest())
    }

    @Test
    fun averageTest() {
        val student = Student("Mako", 70, 100)
        Assertions.assertEquals((70+100)/2, student.getAverage())
    }

    @Test
    fun gradingTest() {
        val student = Student("Mako", 70, 100)
        Assertions.assertEquals('B', student.grading())
    }
}